src/app/shared/components/footer.tsx

```diff
@@ -0,0 +1,54 @@
+import { Typography, Link, Stack } from '@mui/joy';
+
+export const Footer = () => {
+    return (
+        <Stack
+            sx={{
+                pt: 6,
+                pb: 4,
+                px: 0,
```

Comment:
nit:`px: 0` should be the default, so its not needed. prettier accepts both, but for these rather short objects, inlining makes the code a bit more compact 

```tsx
<Stack sx={{ pt: 6, pb: 4, gap: 3}}
````